common/src/main/java/com/radixdlt/utils/UInt64.java

```diff
@@ -118,10 +128,55 @@ private UInt64(Long underlyingValue) {
     this.underlyingValue = Objects.requireNonNull(underlyingValue);
   }
 
+  public UInt64 add(UInt64 other) {
+    return new UInt64(underlyingValue + other.underlyingValue);
```

Comment:
note from author: regular arithmetic for add/sub "just works" with unsigned values